### DISCUSSION TITLE
Extended Statistics are inaccurate from Local and Liveplay

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16833] [Q&A] [answered]
I got a good score. I was recording with OBS (with wrong base canvas, sorry).

This is what I see after I got the play (from the recording):
![Replay 2022-02-09 21-35-45 mp4_snapshot_01 47 010](https://user-images.githubusercontent.com/64581009/153213354-8cc115be-c3d2-4003-8aa0-0974ba2175e3.jpg)

This is after when I watch the replay:
![osu_2022-02-09_21-39-54](https://user-images.githubusercontent.com/64581009/153213408-ec7edb31-d73f-4670-99cc-7a5cb26f7b7c.png)

Score and Timing Distribution isn't looking right. Accuracy heatmap is a little off. Like few dots. 

Is this a bug or is it because of I've played the replay too fast that kinda altered the statistics a bit?

Just in case here's the replay:
[Gumballer playing Machico - fantastic dreamer (Rizia) [Kisses' Insane].zip](https://github.com/ppy/osu/files/8033227/Gumballer.playing.Machico.-.fantastic.dreamer.Rizia.Kisses.Insane.zip)

**Comment by frenzibyte** [comment, marked as answer]
Likely to be you altering the statistics by seeking through the replay in a way that makes it lose precision or otherwise. Eventually osu!lazer will not require you to watch the replay to view score statistics, which is tracked in https://github.com/ppy/osu/issues/14472.

**Reply by NotGumballer91** [reply]
I've tried playing the replay without any speed changes. Same results.
I also tried importing the same replay and the score is increased?
![osu_2022-02-09_22-13-27](https://user-images.githubusercontent.com/64581009/153218715-aae1b66e-3ed1-4d2e-a611-b3b9bb10a20d.png)

But playing the replay gets the same result.

**Reply by NotGumballer91** [reply]
Took my time to play the entire replay in 0.5x speed. Accuracy heatmap is only affected but everything else is still the same.
![osu_2022-02-09_22-23-58](https://user-images.githubusercontent.com/64581009/153220923-b9c81fe9-99da-4d64-80bc-407cc34164c6.png)

I guess I hope it gets consistent once it's implemented

**Reply by bdach** [reply]
> Likely to be you altering the statistics by seeking through the replay in a way that makes it lose precision or otherwise.

That would be a bug.

I'd expect any discrepancies to be only caused by loss of precision at replay record time, not at playback time.

**Reply by integerrr** [reply]
> That would be a bug.

So replay precision should be retained even at scrub speed? If that's the intention then this has been a bug for a long time

**Reply by frenzibyte** [reply]
I would expect cases like https://github.com/ppy/osu/issues/13352 (although this one doesn't have misses) to affect the `HitEvents` list, at playback time, which would happen by seeking through the replay. But as mentioned below, this might as well be relevant to https://github.com/ppy/osu/issues/14573.

**Reply by bdach** [reply]
#13352 would be what I would call a bug yes. The whole reason for the frame-stable playback setup which is in place currently (and will process multiple replay frames per game frame) was to avoid this sort of issue, I believe.

**Comment by integerrr** [comment]
#14573 would be why you're seeing different results even when not seeking the replay

**Comment by peppy** [comment]
why is the first screenshot cut off? can't even see if they are the same play as a result.

accuracy is different so no need to even mention extended stats
